index.html

```diff
@@ -11,7 +11,8 @@
   <body>
     <h1>Testing the Nostr Embed</h1>
 
-    <div id="nostr-embed"></div>
+    <div id="nostr-embed-1"></div>
+    <div id="nostr-embed-2"></div>
```

Comment:
We should probably add some to the readme to explain how embedding works. I'm happy to do that after this PR is merged though.

Reply:
Ok